### PR TITLE
Errors refactoring

### PR DESCRIPTION
**Pull request by seth2810**

### CODE REVIEW

**Comment by seth2810** [comment]
@acusticdemon @vanzay

File: transports/http.js [hunk 1]
```diff
@@ -4,24 +4,31 @@ const {json, send} = micro;
 const cls = require('cls-hooked');
 const uuid = require('uuid');
 const _ = require('lodash');
-const RpcError = require('../errors/RpcError');
+
+const errors = require('../errors');
```
Comment (acusticdemon) [inline transports/http.js]: const {RpcError} = require('../errors');

File: index.js [hunk 2]
```diff
@@ -1,39 +1,42 @@
+const _ = require('lodash');
+
+const errors = require('./errors');
```
Comment (acusticdemon) [inline index.js]: const {RpcError} = require('../errors');

**Comment by seth2810** [comment]
@vanzay /ping

**Review by vanzay — APPROVED** [review]

**Comment by seth2810** [comment]
@acusticdemon I think it's time to release new version

**Review by acusticdemon — APPROVED** [review]

**Pull request merged**